### ISSUE TITLE
Uncaught TypeError: Cannot read property 'split' of undefined

### ISSUE THREAD

**Issue by imdpk** [zeixcom/Anzeixer#5]
I am receiving the following error in my console.

1.3.0 was working fine.

Here seems to be a issue:

```javascript
  /**
   * Get index of the current view size
   *
   * @return {integer} zero-based index of sizes array
   */
  var getSizeIndex = function() {
    return sizes.indexOf(view.split(' ')[0]);
  };
```

**Comment by imdpk** [comment]
The way I resolved this error is by setting a initial view for the global view variable.  It looks like DOMContentLoaded is not updating the view variable fast enough which results is isDesktop, isLandscape etc on page load acting on an undefined variable. I have added a function which just returns the logic to get the current size. I made some few more updates i can request a pull if you would like.

Here is what I have done:

```javascript
  var view = currentView(),
      sizes = ['xs', 's', 'm', 'l', 'xl'];

  function currentView() {
    return window.getComputedStyle(document.querySelector('body'), ':after').getPropertyValue('content').replace(/["']/g, '');
  };
```
